pages/inspire.vue

```diff
@@ -13,3 +13,16 @@
     </v-flex>
   </v-layout>
 </template>
+
+<script lang="ts">
+import { Component, Vue } from 'nuxt-property-decorator'
+
+@Component
+export default class IndexPage extends Vue {
+  head(this: IndexPage): object {
```

Comment:
Doesn't technically break anything, but shouldn't duplicate class names (i.e. this is the "Inspire" page, not "Index")

Reply:
Copy paste mistake.